src/main/java/golf/servletquest/entity/Step.java

```diff
@@ -0,0 +1,51 @@
+package golf.servletquest.entity;
+
+public class Step {
```

Comment:
Can we make all fields final?
[optional]: we can use a builder pattern/ also look please take at lombok annotation @Builder